src/go/rpk/pkg/cli/cmd/topic/config.go

```diff
@@ -58,3 +81,15 @@ func NewSetConfigCommand(
 	}
 	return cmd
 }
+
+// Some config properties are read-only, even if they're not marked as such
```

Comment:
I see that `partition_count` and `replication_factor` are redpanda extensions, why are these not returned as readonly? This doesn't seem like a scalable function into the future.

Reply:
Yeah. I was gonna file an issue for that (I did now: https://github.com/vectorizedio/redpanda/issues/1514) but I figured this could at least unblock some users. However, with the info you shared in [this comment](https://github.com/vectorizedio/redpanda/pull/1512#issuecomment-851679221) I'm not sure it's worth it.